lbry/dht/protocol/routing_table.py

```diff
@@ -225,39 +233,37 @@ def find_close_peers(self, key: bytes, count: typing.Optional[int] = None,
         return []
 
     def get_peer(self, contact_id: bytes) -> 'KademliaPeer':
-        """
-        @raise IndexError: No contact with the specified contact ID is known
-                           by this node
-        """
-        return self.buckets[self.kbucket_index(contact_id)].get_peer(contact_id)
+        return self.buckets[self._kbucket_index(contact_id)].get_peer(contact_id)
 
     def get_refresh_list(self, start_index: int = 0, force: bool = False) -> typing.List[bytes]:
         bucket_index = start_index
         refresh_ids = []
         now = int(self._loop.time())
         for bucket in self.buckets[start_index:]:
             if force or now - bucket.last_accessed >= constants.REFRESH_INTERVAL:
```

Comment:
`last_accessed` doesn't appear do be updated - can it either be fixed (a behavior change probably) or removed (maybe this will reveal side effects that weren't known)